test_wcd2reshare.py

```diff
@@ -110,6 +121,23 @@ def test_fields_of_no_interest_ignored():
     assert wcd2reshare.query_formatter(params) == "type=ISN&lookfor=978-3-16-148410-0"
 
 
+def test_webhook_configures_sentry_if_dsn_present(caplog, monkeypatch):
```

Comment:
Once you move sentry configuration to the top of the module, this test will break because sentry will have already been configured by the time you monkeypatch this env variable in the test. It will get configured as soon as this test module imports `wcd2reshare` and stay configured through all the tests.

To test now you'll need to add `from importlib import reload` to this test module and then within each of the sentry configuration tests reload the `wcd2reshare` import after monkeypatching the env variable, e.g.:

```
    monkeypatch.setenv("SENTRY_DSN", "https://1234567890@00000.ingest.sentry.io/123456")
    reload(wcd2reshare)
    wcd2reshare.lambda_handler({}, {})
```